eth/rpc/backend/backend_suite_test.go

```
@@ -167,24 +169,44 @@
 		case <-ctx.Done():
 			fmt.Println("Timeout reached, transaction not included in a block yet.")
-			return nil, nil
+			return nil, nil, nil
 		default:
 			time.Sleep(1 * time.Second)
 		}
 	}
 }
+
+// broadcastSDKTx broadcasts the given SDK transaction and returns the response
+func (s *BackendSuite) broadcastSDKTx(sdkTx sdk.Tx) *sdk.TxResponse {
```

golangci-lint (1.62.2)

[error] func `(*BackendSuite).broadcastSDKTx` is unused

(unused)